packages/lib/src/components/internal/UIElement/types.ts

```diff
@@ -100,6 +100,12 @@ export type UIElementProps = BaseElementProps &
          * @internal
          */
         paymentMethodType?: string;
+
+        /**
+         * Called when a Component detects, or is told by a SecuredField, that the Enter key has been pressed.
+         * - merchant set config option
+         */
+        onEnterKeyPressed?: (elements: OnKeyPressedObject) => void;
```

Comment:
Minor thing - perhaps we can change the interface of this method to be better aligned with the other callbacks? (We generally pass `component` as last parameter instead of wrapping within an object). Something like:

```js
onEnterKeyPressed?: (activeInputElement: Element, component: UIElement) => void;
```

Reply:
ok